From 816b31257134241008e33dd1bfd9f8ac51f580ff Mon Sep 17 00:00:00 2001
From: Andrey Grodzovsky <Andrey.Grodzovsky@amd.com>
Date: Wed, 15 Jun 2016 14:41:27 -0400
Subject: [PATCH 0567/1722] drm/amdgpu: Add virtual sink to connector for EDID
 override feature.

Virtaul sink will allow to boot without connected display,
store a set mode state and apply it once an actual display
is connected.

Signed-off-by: Andrey Grodzovsky <Andrey.Grodzovsky@amd.com>
Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h b/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h
index 2d02973..fa217346 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h
@@ -550,6 +550,7 @@ struct amdgpu_connector {
 	 * The 'current' sink is in dc_link->sink. */
 	const struct dc_sink *dc_sink;
 	const struct dc_link *dc_link;
+	const struct dc_sink *dc_em_sink;
 	const struct dc_target *target;
 	void *con_priv;
 	bool dac_load_detect;
-- 
2.7.4

